File: app/src/main/java/spryrocks/com/tristimer/presentation/ui/screens/NavigationModel.java
package spryrocks.com.tristimer.presentation.ui.screens;

import com.spryrocks.android.modules.ui.mvvm.model.ObservableField;

public class NavigationModel {
    public final ObservableField<MenuItem> menuItem = new ObservableField<>();

    enum MenuItem {
        TIMER, RESULTS, STATISTICS
    }
}

File: app/src/main/java/spryrocks/com/tristimer/presentation/ui/screens/NavigationModel.kt
package spryrocks.com.tristimer.presentation.ui.screens

import com.spryrocks.android.modules.ui.mvvm.model.ObservableField

class NavigationModel {
    val menuItem : ObservableField<MenuItem> = ObservableField()

    enum class MenuItem {
        TIMER, RESULTS, STATISTICS
    }
}

